app/assets/stylesheets/ideas.scss

```diff
@@ -1,3 +1,10 @@
-// Place all the styles related to the Ideas controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+#deleted {
+  text-align: center;
+  background: #ffe;
```

Comment:
Color literals like `#ffe` should only be used in variable declarations; they should be referred to via variable everywhere else.